### PR TITLE
Release 16.1.0

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #761

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+gallery (16.1.0)
+* Rotate images in the browser [#764](https://github.com/owncloud/gallery/pull/764)
+* Do not break if mount is not available [#739](https://github.com/owncloud/gallery/pull/739)
+* Fix to work properly with 10.0.9 [738](https://github.com/owncloud/gallery/pull/738)
```
Comment (PVince81) [inline CHANGELOG.md]: did you test 16.1.0 with OC 10.0.8 or do we need min-version=10.0.9 ?

Comment (DeepDiver1975) [reply CHANGELOG.md]: yes - tested with 10.0.8

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by phil-davis** [comment]
``appinfo.xml`` has:
```
<version>16.0.2</version>
```

I guess that also should be bumped to ``16.1.0`` ?

File: appinfo/info.xml [hunk 2]
```diff
@@ -29,8 +29,8 @@
 		Compatible with Firefox, Chrome and Internet Explorer 9+
 	</description>
 	<licence>AGPL</licence>
-	<author>Olivier Paroz, Robin Appelman</author>
-	<version>16.0.2</version>
+	<author>Olivier Paroz, Robin Appelman, Thomas Müller</author>
+	<version>16.1.0</version>
```
Comment (DeepDiver1975) [inline appinfo/info.xml:23]: @phil-davis 16.1.0

**Comment by DeepDiver1975** [comment]
[gallery.tar.gz](https://github.com/owncloud/gallery/files/2149761/gallery.tar.gz)

**Pull request merged**